src/components/AppSidebar/AppSidebar.vue

```diff
@@ -522,6 +527,10 @@ $top-buttons-spacing: 6px;
 			// increase the padding to not overlap the menu
 			.app-sidebar-header__desc {
 				padding-right: #{$clickable-area * 2 + $top-buttons-spacing};
+
+				&.app-sidebar-header__desc--without-actions {
+					padding-right: #{$clickable-area + $top-buttons-spacing};
```

Comment:
Why does without actions have more padding than the default ?
The css doesn't make sense there :thinking: 

Without have 50px, default only have 6px ?

Reply:
Well, it's complicated. By default the menu action is part of the flow, so 6px is enough since the action menu itself will push the titles right border to the left. But as soon as you take the menu out of the flow (in case of no figure, or compact mode), the title needs a right-padding. And if there is no menu, the padding must be lower.

Feel free to change the CSS, the new base images in this PR will tell you if the CSS works. 😉